### ISSUE TITLE
Multiprotocol telemetry for Hitec + Graupner HoTT

### ISSUE THREAD

**Issue by mortehoe** [FrSkyRC/ETHOS-Feedback-Community#2745]
- [x] Hitec
- [x] Graupner HoTT

 I see that the telemetry has been enabled for a few protocols but not all ;therefore I am opening this issue for Hitec protocol telemetry.

**Comment by Fordster39** [comment]
Please keep open, I thought it was a 1.5 milestone. ThanksGilSent from my iPadOn Oct 15, 2023, at 9:04 PM, github-actions[bot] ***@***.***> wrote:﻿
This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 30 days.

**Comment by mortehoe** [comment]
Please keep this open.
 it is the only reason I chose Frsky in the first place was the ability to work with Hitec receivers.
I hope this will be in the 1.5 milestone release and not delayed any longer.

**Comment by bsongis-frsky** [comment]
Would you please recommend the needed hardware for Graupner / Hitec as I don't have anything at hand to test!

**Comment by Fordster39** [comment]
I would recommend for the Graupner transmitter an MZ 16 or MZ32, for the receiver with gyro and Hott GR-24 PRO and the non-gryo receiver an GR-24. I cannot make a recommendation on the Hitec equipment.Thanks Gil Wilcox Sent from my iPadOn Nov 23, 2023, at 4:27 AM, Bertrand Songis @ FrSky ***@***.***> wrote:﻿
Would you please recommend the needed hardware for Graupner / Hitec as I don't have anything at hand to test!

**Comment by mortehoe** [comment]
The Hitec Optima 6 and 7 and Hitec minima 6 . This is the Github link https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#HITEC---39
Many thanks

**Comment by krajue** [comment]
I have a Graupner MC20 Hott transmitter, and GR 18+ Vario Stabi receiver for helis.
Telemetry also works perfectly with FRSKY X12S under OpenTX and Edgtx as well as with RM TX16S under EdgeTX.
mfg Jürgen Krause

**Comment by mortehoe** [comment]
I still hope this is going to be released in 1.5.0 as I have heard it is reaching cut off point for this release.

**Comment by it-walter** [comment]
On a TX-16s with Edge-TX (version plays less of a role) and module version 1.3.3.20 or higher, the Graupner LUA script works with all Graupner receivers that I know of.
GR12, GR16, all gyro receivers (GR10C to GR24Pro, Falcon, Hawk, it doesn't matter) can be easily configured via telemetry.
If this were also possible in Ethos, it would be a game changer for many Graupner and EDGE-TX users.

**Comment by mortehoe** [comment]
Ethos is great and the only thing missing ,for me,is telemetry support for Hitec (having multi module support is why I chose Frsky coming from Hitec) but if this is not going to be released soon I am considering going to EdgeTX for full support.
I am still hopeful this will be released in version 1.5.0 and not postponed.

**Comment by bsongis-frsky** [comment]
I have received some hardware to work on Multiplex telemetry. Still nothing for Hitec.

**Comment by mortehoe** [comment]
Sorry, I did not know you required a Hitec receiver hence I only linked the Github multiprotocol link where they had enabled the telemetry.I am happy to send you a Hitec Optima  6 receiver for testing unfortunately I dont have an easily accessible Hitec Minima receiver,would this be enough to help out?
If so where do I send the receiver?

**Comment by bsongis-frsky** [comment]
Any receiver with telemetry will be perfect. It costs less than 2€ to send if you are french and I will return it when I have finished.

**Comment by mortehoe** [comment]
I am in England and happy to send if you can tell me where to send the receiver?
Many thanks in advance.

**Comment by Fordster39** [comment]
Do you need a Hott receiver ? I am in the US but I can send one if needed. Gil Sent from my iPhoneOn Dec 15, 2023, at 3:48 PM, mortehoe ***@***.***> wrote:﻿
I am in England and happy to send if you can tell me where to send the receiver?
Many thanks in advance.

**Comment by mortehoe** [comment]
I am still hoping this will be in the 1.5.0 release and not later.
If you still require the Hitec receiver (assuming no one closer) please can you tell me where to send it?
Many Thanks.

**Comment by bsongis-frsky** [comment]
I will add Multiplex in 1.5.0 then Hitec in 1.5.1

**Comment by erichabg** [comment]
Great, but please don't forget Graupner. 

Am 19. Dezember 2023 14:50:06 MEZ schrieb "Bertrand Songis @ FrSky" ***@***.***>:

**Comment by mortehoe** [comment]
Disappointing since I raised this as an issue in July and the only telemetry to be released is not even in the title.
As stated the only reason I chose Frsky was the Multiprotocol support at the time (including telemetry)I do like Ethos because it is what I would have liked from Opentx with the interface.
Please can you tell me if you still need the Hitec receiver,and if so,where to send it?

**Comment by it-walter** [comment]
Salue Bertrand, it is not clear whether you still need a Graupner receiver for the test or not. I'm from Germany and can lend you one.

**Comment by bsongis-frsky** [comment]
Now I have on my desk one Graupner and one Hitec compatible receiver. So I will work on it as soon as I have finished with 1.5.0 pending issues.

**Comment by Fordster39** [comment]
This is still needed, it was promised for 1.5.0 but got pushed back . Thanks Gil Sent from my iPadOn Apr 3, 2024, at 9:03 PM, github-actions[bot] ***@***.***> wrote:﻿
This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 30 days.

**Comment by mortehoe** [comment]
Responding due to stale message? i am still awaiting this as it is a function and capability under opentx (quickly moved from FRos as not as capable) when I purchased the radio,however I do understand there are more pressing bug fixes caused by 1.5 update; I only wished this had been implemented before 1.5 for Horus users/

**Comment by Flaperon88** [comment]
Hola, i am looking for a fonctional Lua script for Graupner Hott receiver  GR24 ; 24Pro ; GR18 ; GR16 to work on Ethos X20 + Ir4x transmitter.
I would like transfer slowly all my competition model (with specific and fine tune setting) to my X20. During this transfer, i would like keeping my Graupner receiver to have 2 TX options. This will take a lot off time....
I have tried to use the Graupner Hott.lua for opentx without success
I saw some msg, but Is it always in dev ? 
Many thanks

**Comment by Help454** [comment]
Testcenter for Hitec Telemetry is ready :-)
![Hitec Telem](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/165424625/3c6a18e2-0352-4836-ad3d-8a025ba00edd)

**Comment by Flaperon88** [comment]
Nice, don't forget Graupner user.   RX binding OK, but no telemetry.

**Comment by Fordster39** [comment]
Yes, let’s not forget the Graupner user !! GilSent from my iPhoneOn May 14, 2024, at 1:19 PM, Flaperon88 ***@***.***> wrote:﻿
Nice, don't forget Graupner user.   RX binding OK, but no telemetry.

**Comment by digiflo** [comment]
Still waiting for the hott telemetry support too

**Comment by Fordster39** [comment]
Same here still waiting .Sent from my iPhoneOn Jun 28, 2024, at 6:17 PM, digiflo ***@***.***> wrote:﻿
Still waiting for the hott telemetry support too

**Comment by X20-3Q** [comment]
Indeed, it would be nice to have telemetry with the Graupner RX's on Ethos multiprotocol TX's .

**Comment by mortehoe** [comment]
Since most of the major bugs seem to be fixed from version 1.5 release and now in version 1.5.11 we seem to be seeing enhancements, perhaps this issue can now take a higher priority (you hoped to release this fix after 1.5 pending issues.)
I am very keen for this to be released as it was an original function capability when I purchased the X10S.

**Comment by krajue** [comment]
Please make it fixed Graupner Hott Telemetry.
Thanks Juergen

**Comment by Cavalon90** [comment]
Please add HOTT Telemetry. 
If I can do any testing/debugging to speed this up. Please let me know! 
Thanks Chris

**Comment by Joshadam0057** [comment]
Im.not sure.if im.asking.for.the.same.thing but being able.to display graupner t50 esc telemetry to x20 (ethos) would be.nice

**Comment by sbuhre** [comment]
Yes, please add Graupner HOTT telemetry.
Thanks
Stephan

**Comment by mortehoe** [comment]
'I will add Multiplex in 1.5.0 then Hitec in 1.5.1'.
We are now at 1.5.12 and still diminishingly hopeful for a release soon and not at 1.6

**Comment by Help454** [comment]
And please dont forget Hitec

**Comment by krajue** [comment]
'I will add Multiplex in 1.5.0 then Hitec in 1.5.1'.
We are now at 1.5.12 and still diminishingly hopeful for a release soon and not at 1.6
please edit this soon thank you.
Juergen

**Comment by maxpop990** [comment]
Please, i'm standing for Graupner Hott telemetry, many thanks

**Comment by bsongis-frsky** [comment]
Graupner HoTT telemetry first sensors will be available in the next 1.6.0 nightlies

**Comment by mortehoe** [comment]
Any news on Hitec telemetry as this is the only part for me that is missing and the reason I first bought into Frsky coming from Hitec.I love Ethos however Opentx had this function.

**Comment by bsongis-frsky** [comment]
I don't have any Hitec receiver for testing, but I can give it a try!

**Comment by Help454** [comment]
i am waiting for Hitec too

**Comment by mortehoe** [comment]
I thought you had a Hitec compatible receiver (post on jan 4th) if you need a genuine Hitec receiver I am happy to send you one (I live in the UK) if it would help?

**Comment by bsongis** [comment]
I will try to add some Hitec tomorrow, we will see if I need a receiver or if you can do the tests for me!

**Comment by mortehoe** [comment]
Many thanks.

**Comment by bsongis-frsky** [comment]
Hitec added, I cannot test it here, waiting for feedback!

**Comment by Help454** [comment]
i have made litle Test see video
[optima9](https://youtu.be/gmgcmuR5wRI)
[optima9+sensorbox](https://youtu.be/JgC7Tl59gTc)

**Comment by bsongis** [comment]
Which is the sensor plugged to the sensorbox?

**Comment by Help454** [comment]
4x Temp
but with no sensor only with box emergency mode

**Comment by bsongis** [comment]
But the box sends some telemetry, no?

**Comment by Help454** [comment]
no 
the Telemetry you can see is the Basic from Receiver.
For more sensors you need the Sensorbox

**Comment by bsongis** [comment]
In the first video there is no sensor box and no Emergency
In the second video there is a sensor box and Ethos goes to Emergency
I think that some more telemetry is sent when the sensor box is plugged, but it is not correctly parsed.

**Comment by Help454** [comment]
i think so too
i tested a Hott GR 12 Receiver too that looks good
![DSC_0232](https://github.com/user-attachments/assets/a14708cc-f3c2-441f-8773-712fefbba1b6)

![DSC_0233](https://github.com/user-attachments/assets/797c3e02-ffec-45e3-b794-ee9dade19c49)

**Comment by bsongis** [comment]
I see GPS. Is there a GPS sensor? Try to plug the minimum, I would like to know which sensor is wrong

**Comment by Help454** [comment]
There was no GPS sensor pluged not in Hitec and not in Hott

**Comment by bsongis** [comment]
It should be fixed in the next nightlies

**Comment by Help454** [comment]
New Test
Optima 9 receiver with HTS SS Blue and Nitro Sensor Box ist working, no emergency mode.

Telemetry from Optima 9
- [ ] Tx RSSi and TX Quality receive Data but not OK (see first Video)
RX Bat is working

Telemetrie from HTS SS Blue Sensor box
Temp 1,2,3,4 is working
Optical RPM Sensor, working (Throttle 1)
magnetic RPM Sensor, working (Throttle 2)
GPS Satellites, GPS Heading, GPS Alt and GPS Speed is working 
A1 ist working
- [ ] Fuel is receiving data but there is no fuel sensor at "Blue" Sensor box (0%)
- [ ] GPS and GPS clock not working
- [ ] current ist missing (Hitec have two Sensors, 50A and 200A, you must switch in Radio)

Telemetrie from HTS SS nitro Sensor box
Temp 1,2,3,4 is working
Optical RPM Sensor, working (Throttle 1)
magnetic RPM Sensor, working (Throttle 2)
GPS Satellites, GPS Heading, GPS Alt and GPS Speed is working 
- [ ] GPS and GPS clock not working
- [ ] A1 is receiving data but there is no voltage sensor at "Nitro" Sensor box
Fuel is working

can not test Vario, AirSpeed and Servo Manager because i have no Hardware

**Comment by Help454** [comment]
i think the following points will not be a problem because user can del. unused sensors

 HTS SS Blue Sensor box
-Fuel is receiving data but there is no fuel sensor at "Blue" Sensor box (0%) (with Nito SensorBox sensor is working)
 HTS SS nitro Sensor box
-A1 is receiving data but there is no voltage sensor at "Nitro" Sensor box 

tested Optima D and Optima 6 Receiver
-Tx RSSi and TX Quality receive Data but not OK (same like Optima 9)
-RX Bat is working and OK
no more Sensors found, that is OK, because you can not connect Sensor box at Optima 6 and Optima D

**Comment by Help454** [comment]
some Pictures of Optima 9 with Blue Sensor box and maximum of Sensors
![DSC_0234](https://github.com/user-attachments/assets/7a10ff5a-3285-4729-b8d8-2bfb670c08af)
![screenshot optima9 Blue1](https://github.com/user-attachments/assets/015e04eb-755b-48bf-81b9-7bebb99c171b)
![screenshot optima9 Blue2](https://github.com/user-attachments/assets/a55344f9-e590-4ba7-a25a-e58e40c7456d)
![screenshot optima9 Blue3](https://github.com/user-attachments/assets/f34ba7eb-c47c-45e5-ac16-eb61b7de8bc0)

**Comment by Help454** [comment]
some Pictures of Optima 9 with Nitro Sensor box and maximum of Sensors
![DSC_0235](https://github.com/user-attachments/assets/b9b1824c-4254-4458-990a-fc1364a09539)
![screenshot-2024-11-08-76083](https://github.com/user-attachments/assets/ad2f1e3a-2122-4ce4-b374-b6911f6c69af)
![screenshot-2024-11-08-76138](https://github.com/user-attachments/assets/6ee27a7d-f12f-4019-8697-e9972d747ed5)
![screenshot-2024-11-08-76163](https://github.com/user-attachments/assets/7593ebc9-16d0-46fb-9657-3a7f0bc80950)

**Comment by Help454** [comment]
tested newest nightlie
GPS is working now
![screenshot-2024-11-09-49863](https://github.com/user-attachments/assets/32a2b02d-f405-4f4a-bc9a-b826a4283c27)

**Comment by krajue** [comment]
Dear Mr. Songis Bertrand,
could you add Lua for Graupner Hott to make internal settings in the receiver?
Thank you in advance.
Greetings Jürgen

**Comment by bsongis-frsky** [comment]
Graupner Hott / Hitec telemetry should be OK now. Anybody made a test recently? I don't have all receivers / sensors at hand.

**Comment by mortehoe** [comment]
Unfortunately frsky decided to not release version 1.6.0 for my Horus X10 (I had been waiting since version 1.5) therefore the Hitec compatability is no good for me .I am in the process of replacing my Hitec receivers for Elrs receivers hopefully for better support in the future.

**Comment by Hansp26** [comment]
Hello bsongis-frsky

I bought a new Frsky Tandem X18 and use some Graupner Hott receivers, GR-12L ang GR-16. They all work fine with telemetry where "sync" is set. Unfortunatly, i do not see my Variometersensor, which is connected at the "T" port. 
All Graupner Hott receivers are initialized and updated with the latest firmware. The Ethos version is 1.6.2.

**Comment by bsongis** [comment]
Would you do a quick test on Ethos 1.7.0 Nightly, just to see if your sensor is correctly supported?
Please do a backup of your radio first, it's better not to fly with 1.7.0

**Comment by Hansp26** [comment]
Unlucky i cannot find such 1.7.0. I just updated to version 1.6.3, radio and receivers.

**Comment by Wolepo** [comment]
Available here: https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/nightly17

**Comment by Hansp26** [comment]
Ok, i found it and  searched the file(s) i need to upgrade on the radio, but there are many....... 
    On Friday 1 August 2025 at 10:11:52 CEST, Wolepo ***@***.***> wrote:  
 
 Wolepo left a comment (FrSkyRC/ETHOS-Feedback-Community#2745)
Available here: https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/nightly17

**Comment by Hansp26** [comment]
Which file(s) do i need for this upgrade?

Yahoo Mail: Zoeken, organiseren, veroveren 
 
  Op vr, aug 1, 2025 om 10:11 schreef ***@***.***>:   Wolepo left a comment (FrSkyRC/ETHOS-Feedback-Community#2745)
Available here: https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/nightly17

**Comment by Hansp26** [comment]
Hello, i tried  1.7.0 by downloading and installing the frsk-file 1.7.0..Ethos showed me the update, but after shutdown the bootloader and restart the radio, Ethos won't start. I was a little shocked, reinstalled 1.6.3 wherafter the radio started again. I restored the backup ,so my radio is uptodate again. Uunlucky, i could not test 1.7.0.

Yahoo Mail: Zoeken, organiseren, veroveren 
 
  Op vr, aug 1, 2025 om 10:11 schreef ***@***.***>:   Wolepo left a comment (FrSkyRC/ETHOS-Feedback-Community#2745)
Available here: https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/nightly17
